tenable/sc/organization.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Please add in the restructured text docstring.  Also please use single quotes for consistency.

Reply:
done